src/Controller/UserController.php

```diff
@@ -35,35 +37,43 @@ public function index()
      *
      * @Route("/users/new")
      * @Method("POST")
-     * Method will used for creating the new user from REST
+     * Method will used for creating the new user
      */
 
     public function newUserAction(Request $request)
     {
+        $educationArray = [
+            
+            'UG College' => 'UG College',
+            'PG College' => 'PG College',
+            'Masters College' => 'Masters College',
+            'Others College' => 'Others College',
+       ];
+
         $form = $this->createFormBuilder()
             ->add('userFirstName', TextType::class, array(
-                'constraints' => new NotBlank(),
+                'constraints' => new NotBlank(), 'attr' => array('class' => 'form-control')
             ))
             ->add('userLastName', TextType::class, array(
-                'constraints' => new NotBlank(),
+                'constraints' => new NotBlank(), 'attr' => array('class' => 'form-control')
             ))
             ->add('userEmail', TextType::class, array(
-                'constraints' => new NotBlank(),
+                'constraints' => new NotBlank(), 'attr' => array('class' => 'form-control')
```

Comment:
all form validations has to be separate..

Reply:
i.e validations should be in yml file

Reply:
I have handled the form validations in yml file.